emperor/core.py

```diff
@@ -244,6 +247,34 @@ def _repr_html_(self):
 
         return display(HTML(str(self)))
 
+    def _validate_metadata(self, ignore_missing_samples):
+        difference = set(self.ordination.samples.index) - set(self.mf.index)
+
+        if len(difference) == len(self.ordination.samples):
```

Comment:
I don't really understand this comparison or the error, can you check them?

Reply:
After looking at this comparison, I've changed this to compare the sample names themselves instead of the length.

This error occurs when a user accidentally mismatches the metadata and the coordinates, if you have any suggestions to make the error message clearer that would help a lot.